src/components/binding/binding.js

```diff
@@ -150,13 +162,16 @@ genotet.BindingView = function(viewName, params) {
     .on('genotet.gene', function(event, data) {
       this.data.tracks[data.trackIndex].fileName = data.fileName;
       this.loader.loadFullTrack(data.trackIndex, data.fileName,
-        this.data.chr);
+        this.data.chr, false);
     }.bind(this));
 
   $(this.loader)
     .on('genotet.chr', function(event, chr) {
       this.panel.updateChr(chr);
     }.bind(this))
+    .on('genotet.addTrack', function() {
+      this.loader.loadBindingList();
+    }.bind(this))
     .on('genotet.track', function() {
```

Comment:
Track is now a confusing name. Use updateTrack or similar to describe this message as the user changes tracks in the panel.